src/components/comments/LikeDislikeComment.jsx

```diff
@@ -0,0 +1,176 @@
+import {Comment, Icon, Label} from "semantic-ui-react";
+import React, {Component} from "react";
+import axios from "axios";
+import PropTypes from 'prop-types';
+import {
+    get_axios_config,
+    URL
+} from "../../store/actions/async/CommentActions";
+import { infoToast } from '../UI/Toast/toast';
+
+
+class LikeDislikeComment extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            commentId: null,
+            likeStatus: false,
+            dislikeStatus: false,
+            likes: 0,
+            dislikes: 0
+        };
+    }
+
+    componentDidMount(){
+        const {
+            id,
+            liked,
+            disliked,
+            likeCount,
+            dislikeCount
+        } = this.props;
+
+        this.setState({
+            commentId: id,
+            likeStatus: liked,
+            dislikeStatus: disliked,
+            likes: likeCount,
+            dislikes: dislikeCount
+        })
+    }
+
+    likeComment = () => {
+        const { commentId, likeStatus } = this.state;
+        const { isOwner } = this.props;
+        if (!localStorage.getItem("token")){
+            infoToast("Please log in to like a comment")
+        }else if(isOwner){
+            infoToast("You cannot like your own comment!")
+        }else if(!likeStatus){
+            try{
+                axios.post(`${URL}/articles/comments/${commentId}/like`,
+                {}, get_axios_config())
```

Comment:
@pbkabali good work on the implementation however I think earlier as mention by @Karuhanga  I think this is not the best place to make api calls otherwise good work